test/test-network/meson.build

```diff
@@ -0,0 +1,188 @@
+        conf/25-tunnel-local-any.network
+        conf/25-tunnel-remote-any.network
+        conf/25-tunnel.network
+        conf/25-vcan.netdev
+        conf/25-veth.netdev
+        conf/25-vrf.netdev
+        conf/25-vrf.network
+        conf/25-vti-tunnel-local-any.netdev
+        conf/25-vti-tunnel-remote-any.netdev
+        conf/25-vti-tunnel.netdev
+        conf/25-vti6-tunnel-local-any.netdev
+        conf/25-vti6-tunnel-remote-any.netdev
+        conf/25-vti6-tunnel.netdev
+        conf/25-vxcan.netdev
+        conf/25-vxlan.netdev
+        conf/25-wireguard-23-peers.netdev
+        conf/25-wireguard-23-peers.network
+        conf/25-wireguard-preshared-key.txt
+        conf/25-wireguard-private-key.txt
+        conf/25-wireguard.netdev
+        conf/25-wireguard.netdev.d/peer.conf
+        conf/25-wireguard.network
+        conf/26-bridge-slave-interface-1.network
+        conf/26-bridge-slave-interface-2.network
+        conf/26-bridge.netdev
+        conf/6rd.network
+        conf/bond-slave.network
+        conf/bond99.network
+        conf/bridge99-ignore-carrier-loss.network
+        conf/bridge99.network
+        conf/configure-without-carrier.network
+        conf/dhcp-client-anonymize.network
+        conf/dhcp-client-critical-connection.network
+        conf/dhcp-client-gateway-onlink-implicit.network
+        conf/dhcp-client-ipv4-dhcp-settings.network
+        conf/dhcp-client-ipv4-only-ipv6-disabled.network
+        conf/dhcp-client-ipv4-only.network
+        conf/dhcp-client-ipv6-only.network
+        conf/dhcp-client-ipv6-rapid-commit.network
+        conf/dhcp-client-listen-port.network
+        conf/dhcp-client-route-metric.network
+        conf/dhcp-client-route-table.network
+        conf/dhcp-client-timezone-router.network
+        conf/dhcp-client-vrf.network
+        conf/dhcp-client-with-ipv4ll-fallback-with-dhcp-server.network
+        conf/dhcp-client-with-ipv4ll-fallback-without-dhcp-server.network
+        conf/dhcp-client.network
+        conf/dhcp-server-timezone-router.network
+        conf/dhcp-server-veth-peer.network
+        conf/dhcp-server.network
+        conf/dhcp-v4-server-veth-peer.network
+        conf/erspan.network
+        conf/gretap.network
+        conf/gretun.network
+        conf/ip6gretap.network
+        conf/ip6gretun.network
+        conf/ip6tnl.network
+        conf/ipip.network
+        conf/ipv6-prefix-veth.network
+        conf/ipv6-prefix.network
+        conf/ipvlan.network
+        conf/ipvtap.network
+        conf/isatap.network
+        conf/macsec.network
+        conf/macvlan.network
+        conf/macvtap.network
+        conf/netdev-link-local-addressing-yes.network
+        conf/routing-policy-rule-dummy98.network
+        conf/routing-policy-rule-test1.network
+        conf/sit.network
+        conf/vlan6.netdev
+        conf/vlan6.network
+        conf/vti.network
+        conf/vti6.network
+        conf/vxlan-test1.network
+        conf/vxlan.network
+'''.split()
```

Comment:
We're going to suffer here. I think this should be automatic and everything under conf/ should be installed.